Modules/Office365DSC/Modules/Office365DSCUtil.psm1

```diff
@@ -1715,13 +1715,33 @@ function Get-UsersLicences
         $GlobalAdminAccount
     )
     Test-O365ServiceConnection -GlobalAdminAccount $GlobalAdminAccount
-    Write-Verbose -Message "Store all users licences information in Global Variable for futur usage."
+
+    Write-Verbose "Store all users licenses information in Global Variable for future usage."
+
     #Store information to be able to check later if the users is correctly licences for features.
-    if ($null -eq $Global:UsersLicences)
+    if ($null -eq $Global:UsersLicenses)
     {
-        $Global:UsersLicences = Get-MsolUser -All | Select-Object UserPrincipalName, isLicensed, Licenses
+<#         $Global:UsersLicenses = @()
```

Comment:
Why are we changing the logic here?

Reply:
The idea was to move away from using MSOL, but IIRC I found this challenging to do as the output isn't the same using AzureAD, hence the convoluted logic. WDYT?

Reply:
I say we leave it as is for the moment. Remove the comments and we should be all go to go. Will do a final round of integration tests once you removed them.

Reply:
OK done.